Equilobe.DailyReport.BL/Jira/JiraApiUrls.cs

```diff
@@ -58,9 +58,9 @@ public static string Search(string jql)
             return string.Format("rest/api/2/search?jql={0}", jql);
         }
 
-        public static string ResolvedIssues(string fromDate, string endDate)
+        public static string ResolvedIssues(string projectKey, string fromDate, string endDate)
         {
-            return string.Format("statusCategory = 'Done' AND resolved >= '{0}' AND resolved <= '{1}'", fromDate, endDate);
+            return string.Format("statusCategory = 'Done' AND resolved >= '{0}' AND resolved <= '{1}' AND project={2}", fromDate, endDate, projectKey);
```

Comment:
E un filtru in plus. Pana acum nu am avut probleme, dar daca un user ponteaza in mai multe proiecte o sa apara toate log-urile. Si am dat aici commit-ul ca tot lucram in branch. Daca nu e ok dau revert si il dau in master sau in alta parte.